R/crm_text.R

```diff
@@ -124,10 +127,8 @@
 #' dois <- out$data$DOI[1:10]
 #' # res <- list()
 #' # for (i in seq_along(dois)) {
-#' #   tmp <- crm_links(dois[i], "all")
-#' #   tmp <- setNames(tmp, "pdf")
-#' #   attr(tmp, "type") <- "pdf"
-#' #   res[[i]] <- crm_text(tmp, type = "pdf", cache=FALSE)
+#' # tmp <- crm_links(dois[i], "all")
+#' # res[[i]] <- crm_text(tmp, type = "pdf", cache=FALSE, overwriteUnspecified=T)
```

Comment:
please do use `TRUE`/`FALSE` instead of `T`/`F` throughout - just go to next line if greater than 80 width